### PR TITLE
Fix MDNS vendor / add TimedRequest support

### PR DESCRIPTION
**Pull request by mfucci**
Required for iOS / Android support

### CODE REVIEW

**Review by Apollon77 — APPROVED** [review]
Will check specs tomorrow for the one type but rest looks ok.

File: src/matter/interaction/InteractionMessages.ts [hunk 1]
```diff
@@ -144,3 +144,7 @@ export const TlvInvokeResponse = TlvObject({
     }))),
     interactionModelRevision: TlvField(0xFF, TlvUInt8),
 });
+
+export const TlvTimedRequest = TlvObject({
```
Comment (Apollon77) [inline src/matter/interaction/InteractionMessages.ts:148]: Spec reference Core Spec 10.6.8

**Review by Apollon77 — APPROVED** [review]

File: src/matter/interaction/InteractionMessages.ts (hunk 1 above)
Comment (mfucci) [reply src/matter/interaction/InteractionMessages.ts:151]: Added!

**Comment by mfucci** [comment]
Need approval again after adding spec ref

**Review by Apollon77 — APPROVED** [review]

**Pull request merged**